ds0_utils.py

```diff
@@ -75,3 +81,38 @@ def required_commit():
 
     if options and options.forcecommit:
         return options.forcecommit
+
+def repository():
+    options = parse_ec2_userdata()
+
+    if options and options.repository:
+        if ':' in options.repository:
+            (origin, branch) = options.repository.split(':')
+        else:
+            origin = options.repository
+            branch = ''
+        return { 'origin': origin, 'branch': branch }
+    else:
+        return { 'origin': '', 'branch': '' }
+
+def disable_commit_verification():
+    options = parse_ec2_userdata()
+
+    if options and options.disablecommitverification:
+        return options.disablecommitverification
+    else:
+        return False
+
+def allowed_keys():
+    options = parse_ec2_userdata()
+
+    if options and options.allowedkeys:
+        return options.allowedkeys
+    else:
+        default_key = {
+            'id': '7123CDFD',
+            'file': '/home/ubuntu/datastax_ami/repo_keys/DataStax_AMI.7123CDFD.key',
```

Comment:
Using a hardcoded path is what I've been doing so far and would ideally be accepted here.

I chose the repo key naming convention to be able to easily jog my memory as to why we store certain keys as well as documenting the keys for the community. If neither of you two think there's an issue with changing the naming format, I'd be open to the change.